lib/usdUfe/ufe/UsdClipboardCommands.cpp

```diff
@@ -320,9 +321,12 @@ void UsdCutClipboardCommand::execute()
     const auto& sceneItemOpsHandler
         = Ufe::RunTimeMgr::instance().sceneItemOpsHandler(UsdUfe::getUsdRunTimeId());
 
-    // Step 2. Delete the selected items.
+    auto ufeSn = Ufe::GlobalSelection::get();
+
+    // Step 2. Deselect then delete the selected items.
     for (auto&& item : _selection) {
         if (auto usdItem = downcast(item)) {
+            Ufe::SelectionRemoveItem::createAndExecute(ufeSn, item);
```

Comment:
Fixed a bug with the cut command not actually removing the cut item from the selection. The Ufe global selection still contained the path of the cut prim. If you asked Maya for the Ufe selection it would return empty string because it could not create an item from the path (since the prim is actually deleted). But the Ufe selection still contained that one invalid path.